src/enums/filingStatus.ts

```diff
@@ -11,4 +11,5 @@ export enum FilingStatus {
   PENDING = 'PENDING',
   PENDING_CORRECTION = 'PENDING_CORRECTION',
   WITHDRAWN = 'WITHDRAWN',
+  AFFILIATION_INVITATION_PENDING = 'AFFILIATION_INVITATION_PENDING',
```

Comment:
Updated, will check if the invitation details `id` is set in the todo item.

Reply:
But `id` may not be unique to affiliation invitations...

Do you want me to add a value to the Filing Types enum? (AFFILIATION_INVITATION)

Reply:
I added `affiliationInvitationDetails` in the todo item:
So I am checking this: `return !!item.affiliationInvitationDetails?.id`

I wanted to avoid adding AFFILIATOIN_INVITATION as filing type because it is not a filing, like you know. Though it is requested to be displayed along filing todos. 

Is the check I am doing good enough (that part should be unique to the affiliation invitation)

Reply:
Oh I see -- `id` is a property in a sub-object.

Wouldn't it be easier/cleaner to check item.name? I'm OK with adding it to the filing type because it's kind of handled like a filing/task.

Or, why not just check if item.affiliationInvitationDetails is not falsy?

Reply:
Yeah, it makes sense, will update to check `affiliationInvitationDetails` is not falsy